screens/feed/FeedScreen.tsx

```diff
@@ -71,7 +71,7 @@ const FeedScreen: React.FC = () => {
   };
 
   if (error) return <Error message={error.message} apolloError={error} />;
-  if (loading) return <Loading />;
```

Comment:
Maybe put the loading component in the footer option in the flatlist, like in ongoingchallengesscreen.tsx

Reply:
Yes, can do that.